modules/ROOT/pages/upgrade-policies-proxies.adoc

```diff
@@ -0,0 +1,128 @@
+ .. Go to *Policies*.
+ .. In *Applied policy status*, verify that the latest definition is applied.
+ * If the latest definition is applied, click on the three dots menu and select *Check for implementation updates*.
+ * If the latest definition is not applied, update to the latest definition.
+
+[[upgrading-api-level-policies]]
+== Upgrading API-Level Policies
+
+Upgrade your API-level policies (included and custom) to Java 17.
+
+[[upgrading-included-api-level-policies]]
+=== Upgrading Included API-Level Policies
+
+To upgrade xref:gateway::policies/policies-availability-by-gateway.adoc[MuleSoft included policies] that are API-level to Java 17:
+
+. In API Manager, go to *API Administration > <API Name> > Policies > Add a policy* to xref:gateway::policies-included-apply.adoc[apply policies] at the individual API level. 
+. Upgrade each of your API-level policies:
+.. In *Policy version*, select the latest version of the included policy.
+.. In *Method & resource conditions*, select either *Apply configuration to all API method & resources* or *Apply configuration to specific API method & resources*.
+.. Click *Apply*.
+
+[[upgrading-custom-api-level-policies]]
+=== Upgrading Custom API-level Policies
+
+To upgrade custom API-level policies to Java 17:
+
+. In API Manager, go to *API Administration > <API Name> > Policies > Add a policy* to xref:gateway::policies-included-apply.adoc[apply policies] at the individual API level. 
+. Upgrade the required extensions and connectors to Java 17.
+ * If your custom policy has Anypoint Connectors, refer to xref:java-support.adoc#anypoint-connectors-and-modules[Upgrading Anypoint Connectors and Modules].
+ * If your custom policy has custom connectors, refer to <<Upgrading Custom Connectors INSERT LINK>>.
+. xref:gateway::policies-custom-upgrade.adoc[Upgrade the custom policy (link to new page in Gateway)].
+. xref:gateway::policies-custom-package.adoc[Package the custom policy].
+. xref:gateway::policies-custom-upload-to-exchange.adoc[Republish the custom policy to Exchange].
+. Apply the new version of the custom policy that is compatible with Java 17 to the API instance:
+ .. Go to *Policies*.
+ .. In *Applied policy status*, verify that the latest definition is applied.
+ * If the latest definition is applied, click on the three dots menu and select *Check for implementation updates*.
+ * If the latest definition is not applied, update to the latest definition.
+
+[[upgrading-api-proxies]]
+== Upgrading API Proxies
+
+The steps to upgrade API proxies to Java 17 are a little different, depending on which deployment model you use. 
+
+* If you use Basic endpoint, deploy the adapted application to the server from Mule runtime and connect it to API Manager using autodiscovery. For more information, refer to xref:java-support.adoc#mule-runtime[].
+* If you use a Basic endpoint API instance to update your instance, update the Mule application connecting to your API instance.
+
+[IMPORTANT]
```

Comment:
This note should be moved to the top so they see it before they start upgrading proxies.